docs/changelog.md

```diff
@@ -47,6 +50,7 @@ permalink: /changelog/
 
 * **feature-prompts**, **browser-engine-gecko***
   * Added support for Authentication dialogs.
+  * Added support for [datetime-local](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/datetime-local) and [time](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/time) pickers.
```

Comment:
you duplicated this changelog entry. ;-)

Reply:
Oops 🙃  Thanks for noticing it :)

Reply:
Fixed!